src/main/scala/is/hail/HailContext.scala

```diff
@@ -249,12 +249,6 @@ class HailContext private(val sc: SparkContext,
     if (inputs.isEmpty)
       fatal("arguments refer to no files")
 
-    inputs.foreach { input =>
```

Comment:
could be in a different PR, I guess. This prevents loading .gen.gz files, which are common.